src/spikeinterface/core/waveform_extractor.py

```diff
@@ -1457,16 +1469,17 @@ def extract_waveforms(recording, sorting, folder=None,
 extract_waveforms.__doc__ = extract_waveforms.__doc__.format(_sparsity_doc, _shared_job_kwargs_doc)
 
 
-def load_waveforms(folder, with_recording=True, sorting=None):
+def load_waveforms(folder, with_recording: bool = True, sorting: Optional[BaseSorting] = None) -> WaveformExtractor:
     """
     Load a waveform extractor object from disk.
 
     Parameters
     ----------
     folder : str or Path
         The folder / zarr folder where the waveform extractor is stored
-    with_recording : bool, optional
-        If True, the recording is loaded, by default True
+    with_recording : bool | BaseRecording, optional
```

Comment:
Should't the BaseRecording be removed here?

Reply:
Done :)